### DISCUSSION TITLE
Cannot import replays into lazer on android

### DISCUSSION THREAD

**Discussion by Mrcubix** [ppy/osu#15345] [Q&A]
Like the title state, replays cannot be imported on android as it will state that it couldn't find an application that can open the replay file.

This is probably due to lack of file association when the PR for android import support was merged.

EDIT: Here is a screenshot of the message in question in case anyone ask

![20211028_135838](https://user-images.githubusercontent.com/39861216/139251670-5af38106-04f2-42c6-8cb5-bb5a6ac0ae7e.jpg)

PS: maps and skins work.

OS: Android 8.0
Build: 2021.1028.0

**Comment by frenzibyte** [comment]
Should be resolved in next release with #15403.

**Reply by Mrcubix** [reply]
Seem like every types of file association on android has been removed, 
reboot doesn't fix it
&
uninstall, reboot, clean install still doesn't work 
&
 uninstall, shutdown, boot, clean install doesn't work either

**Reply by gagahpangeran** [reply]
The file association still working on my phone.

**Reply by Mrcubix** [reply]
File association disappeared in android settings after uninstall and are not set at the next install.
The use of a 3rd party file manager or use of the option `Share` is now the only way to import files into lazer.

Share:
![20211103_170400](https://user-images.githubusercontent.com/39861216/140097699-97f01dc0-7a11-41b9-bf6c-51332faf5265.jpg)

3rd part file manager:
![20211103_170438](https://user-images.githubusercontent.com/39861216/140097721-1a46a9ba-fb3f-4aba-bc3f-bd7ebb967d94.jpg)

**Reply by peppy** [reply]
Nothing has changed from our end. This is likely something with your device configuration.
